akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowGroupedWeightedSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.scaladsl
+
+import akka.stream.testkit.{ ScriptedTest, StreamSpec, TestPublisher, TestSubscriber }
+import akka.testkit.TimingTest
+import akka.util.unused
+
+import scala.collection.immutable
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class FlowGroupedWeightedSpec extends StreamSpec("""
+    akka.stream.materializer.initial-input-buffer-size = 2
+  """) with ScriptedTest {
+
+  "A GroupedWeighted" must {
+    "produce empty sequence (group) when source is empty" in {
```

Comment:
looks more like it verifies no group than empty group to me, so update test name accordingly